scripts/build.sh

```diff
@@ -7,22 +7,22 @@ BUILD_CMD="python setup.py bdist_egg bdist_wheel upload"
 pushd src/
 
     pushd mercury-common
-    tox
+    tox -r
```

Comment:
As rebuild can take quite a while; it'd probably be helpful to have an option to the script to set the `-r` into a `TOX_OPTIONS` variable that is then used.

perhaps a `--no-rebuild` flag:
```bash
# default to rebuild
OPTION_TOX_REBUILD="yes"
for ARG in $@
do
     if [ "${ARG}" == "--no-rebuild" ]; then
         OPTION_TOX_REBUILD="no"
     fi
done
TOX_OPTIONS = ""
if [ "${OPTION_TOX_REBUILD}" == "yes" ]; then
   TOX_OPTIONS="${TOX_OPTIONS} -r"
fi
```
$0.02 - not a blocker

Reply:
With python3.6 and wheels, it takes six seconds longer on my mac (per package). The reason I am adding it is to bring in new deps, as it didn't seem like requirements.txt or setup.py was being run unless I used the -r switch. It's not necessary to run every time, so I'll do as you suggested and add an option to scripts

Reply:
Correct - the package is only reinstalled if `-r` is specified.

Some of my tox environments have taken a few minutes to run; why I don't know. Might be VPN or something.

Thanks!